### PR TITLE
Ansible-based installation reuses incorrect/unintended configuration value

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1577220

### CODE REVIEW

**Comment by mburke5678** [comment]
@cevich PTAL

**Comment by cevich** [comment]
Thanks, LGTM[*]

*: I am not an English major :grinning:

**Comment by mburke5678** [comment]
@jianlinliu PTAL

**Comment by jianlinliu** [comment]
LGTM.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -2521,6 +2521,15 @@ If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[CAUTION]
+====
+The installer caches playbook configuration values for 10 minutes, by default. If you change any system, network, or inventory configuration,
```
Comment (kalexand-rh) [inline install_config/install/advanced_install.adoc]: If you can't change the setting, I'd remove ", by default"

s/configuration, then/configuration and then

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -2521,6 +2521,15 @@ If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[CAUTION]
+====
+The installer caches playbook configuration values for 10 minutes, by default. If you change any system, network, or inventory configuration, 
+then re-run the installer within that 10 minute period, the new values are not used and the previous values are used instead.
```
Comment (kalexand-rh) [inline install_config/install/advanced_install.adoc]: s/used/used or s/used and the/used. The

**Review by kalexand-rh — COMMENTED** [review]
Couple of nitpicks, but this sounds good. :)

File: install_config/install/advanced_install.adoc (hunk 2 above)
Comment (mburke5678) [reply install_config/install/advanced_install.adoc]: @kalexand-rh You are suggesting:
_...the new values are not used._ or
_...the new values are not used. The previous values are used instead. _
??

Comment (kalexand-rh) [reply install_config/install/advanced_install.adoc]: @mburke5678, I see my typo. You either need a comma after used or to split the sentence. My apologies.

**Comment by mburke5678** [comment]
@cevich @jianlinliu What OpenShift versions does this apply to? Is this slated for 3.10?

**Comment by jianlinliu** [comment]
As far as I know, this is only relative to ansible.

**Review by sdodson — APPROVED** [review]
LGTM, not sure if this warrants CAUTION or a less severe callout. I think the chances of customers running into this problem are very very low.

**Comment by sdodson** [comment]
> @cevich @jianlinliu What OpenShift versions does this apply to? It looks like fact caching has been around since 3.2...

Yes, ansible caches facts by default. Relevant to all versions.

**Comment by mburke5678** [comment]
@sdodson Is I changed the admonition to Warning

**Review by sdodson — APPROVED** [review]

**Pull request merged**